Intel/AlderLake.cs

```diff
@@ -497,24 +390,25 @@ public string GetHelpText()
 
             private string[] computeMetrics(string label, NormalizedCoreCounterData counterData)
             {
+                float slots = counterData.activeCycles * coreType.AllocWidth;
```

Comment:
Alder Lake (technically Ice Lake and later) has a new fixed counter, TOPDOWN.SLOTS (IA32_FIXED_CTR3, MSR 0x30C), and the SDM suggests using that as the denominator for topdown events.  Is it worth using that, or is is deriving that from active cycles good enough?

I don't know if there's a difference since I don't have an Ice Lake or newer Intel CPU to test with